lib/main/util/robot.ts

```diff
@@ -0,0 +1,83 @@
+import cmd from 'node-cmd'
+import { desktopCapturer } from 'electron'
+import psList from 'ps-list'
+import robot from 'robotjs'
+import Store from 'electron-store'
+
+const store = new Store()
+
+class Robot {
+  async trigger(id: string) {
+    const sequence = store.get(id) as Sequence
+    if (sequence.openedApp) {
```

Comment:
Add a comment above this line about what the if statement checks/what it's doing. Sentence structure usually beginning with a capitalized verb.